packages/terra-form-input/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Added onInput prop.
```

Comment:
It would be helpful to add a little more detail about this prop and what it does in the changelog

Reply:
Updated here https://github.com/cerner/terra-core/pull/3135/commits/17a5bc400687a7f9b1b945eb45b51f42adbb27f6